src/main/java/org/apache/maven/it/Embedded3xLauncher.java

```diff
@@ -268,4 +271,26 @@ public String getMavenVersion()
         throw new LauncherException( "Could not determine embedded Maven version" );
     }
 
+    /**
+     * Replicates the logic from <a href="https://git.io/JSMug">Maven start script</a> to find 
+     * the project's base directory.
+     * @param workingDirectory the working directory
+     * @return the project's base directory
+     */
+    private static File findProjectBaseDirectory( File workingDirectory )
+    {
+        File folder = workingDirectory;
```

Comment:
directory, *not* folder

Reply:
fixed in https://github.com/apache/maven-verifier/pull/6/commits/1a232adec518cfba6d63fb742b63ac50b1d868e7.